src/libstd/primitive_docs.rs

```diff
@@ -566,7 +566,9 @@ mod prim_array { }
 #[doc(alias = "[")]
 #[doc(alias = "]")]
 #[doc(alias = "[]")]
-/// A dynamically-sized view into a contiguous sequence, `[T]`.
+/// A dynamically-sized view into a contiguous sequence, `[T]`. Contiguous here
+/// means that elements are layed out so that every element is the same
```

Comment:
s/layed out/laid out/

Had to look this one up myself: https://writingexplained.org/laid-out-or-layed-out